src/Symfony/Component/HttpKernel/Tests/Exception/BadGatewayExceptionTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Symfony\Component\HttpKernel\Tests\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadGatewayException;
+
+class BadGatewayExceptionTest extends HttpExceptionTest
+{
+    protected function createException()
+    {
+        return new BadGatewayException();
```

Comment:
If existing tests do not test the codes, maybe they should be added instead of omitting it in new tests as well (not the scope of this PR though)